forge16/src/main/java/com/envyful/poke/removal/forge/config/PokeRemovalConfig.java

```diff
@@ -237,11 +236,13 @@ public enum RemovalType {
 
             return true;
         }, (types, entity) -> {
-            String entityString = entity.getType().getRegistryName().toString();
+            if(entity.getType().getRegistryName() != null){
```

Comment:
Please follow the formatting with spacing correctly

Reply:
Not 100% sure if managed to resolve this properly can you please confirm that?